eclass/eutils.eclass

```diff
@@ -5,6 +5,7 @@
 # @MAINTAINER:
 # base-system@gentoo.org
 # @SUPPORTED_EAPIS: 5 6 7
+# @PROVIDES: desktop edos2unix epatch estack ltprune multilib preserve-libs strip-linguas toolchain-funcs vcs-clean wrapper
```

Comment:
I've decided to list them all since we ban eutils in EAPI 8 anyway, and otherwise this causes a lot of noise.